### DISCUSSION TITLE
Strategy userID

### DISCUSSION THREAD

**Discussion by edilsonnewbit** [Unleash/unleash#3132] [General]
In cases where I need to release the feature for a large list of user IDs, for example two million users, how would this be done in Unleash? Is the storage done in a comma-separated string?

**Comment by sighphyre** [comment]
Hi @edilsonnewbit, we recommend not using such a large list of user IDs in Unleash, since this will have a very painful impact on the performance of both your SDK and network. Usually the approach we would take here is to store your user IDs in something like Redis, then query Redis to check if your user is in that list and pass that true or false as a context property and use a strategy constraint on the toggle. We have a very small sample here: https://github.com/sighphyre/UnleashAndRedis where we're using a strategy constraint to achieve what you're describing. That's absolutely not production ready, it's just to demonstrate the idea
